packages/stark-core/src/modules/session/services/session.service.ts

```diff
@@ -358,7 +358,12 @@ export class StarkSessionServiceImpl implements StarkSessionService {
 			this._devAuthenticationHeaders = new Map<string, string>();
 		}
 
-		devAuthenticationHeaders.forEach((value: string, key: string) => this._devAuthenticationHeaders.set(key, value));
+		devAuthenticationHeaders.forEach((value: string, key: string) => {
```

Comment:
Should not we change this to `(value: string | string[], key: string) => {`?

Because you mention in the comment above that Angular supports both but it is not implemented in this case...

Reply:
Indeed, I'll fix this